src/mdfactory.h

```diff
@@ -24,15 +24,28 @@ class MDfactory
   public:
     static CabanaMD *create( InputCL commandline )
     {
+        // If device is not specifically enabled, choose in this order
         int device = commandline.device_type;
+        if ( device == NONE )
```

Comment:
I think this is still worth adding and picking up the kokkos default execution space

Reply:
I opened #75 instead